test/components/Frame.js

```diff
@@ -14,33 +10,23 @@ css.global('*, *:before, *:after', { boxSizing: 'inherit' })
 
 css.global('body', {
   width: '100%',
-  fontFamily: fontFamilies.sansSerifRegular
+  fontFamily: fontFamilies.sansSerifRegular,
 })
 
 // avoid gray rects over links and icons on iOS
 css.global('*', {
-  WebkitTapHighlightColor: 'transparent'
+  WebkitTapHighlightColor: 'transparent',
 })
 // avoid orange highlight, observed around full screen gallery, on Android
 css.global('div:focus', {
-  outline: 'none'
+  outline: 'none',
 })
 
-
-const Index = ({
-  children,
-  dark
-}) => (
-  <ColorContext.Provider value={dark && colors.negative}>
-    {dark && (
-      <style
-        dangerouslySetInnerHTML={{
-          __html: `html, body { background-color: ${colors.negative.containerBg}; color: ${colors.negative.text}; }`
-        }}
-      />
-    )}
+// 'light' or 'dark' modes are available
+const Index = ({ children }) => (
+  <ColorContextProvider root colorSchemeKey='dark'>
```

Comment:
Wouldn't this change always change the ColorContext to "dark", whereas before it was only if the dark prop was provided? Don't have all the context though, so might just not understand what's going on.

Reply:
you, my friend, are very much correct. thanks!